rust/types/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bpx-api-types"
-version = "0.5.1"
+version = "0.6.2"
```

Comment:
@tomlinton Do we have a versioning guideline for API types?

Reply:
Not currently